examples/minimal/.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2
+
+references:
+  set_environment_variables: &set_environment_variables
+    run:
+      name: Set Environment Variables
+      command: |
+        echo 'export CI_SHA1=$CIRCLE_SHA1' >> ${BASH_ENV}
+        echo 'export CI_BRANCH=$CIRCLE_BRANCH' >> ${BASH_ENV}
+        echo 'export CI_BUILD_NUM=$CIRCLE_BUILD_NUM' >> ${BASH_ENV}
+        echo 'export CI_TAG=$CIRCLE_TAG' >> ${BASH_ENV}
+        echo 'export ROK8S_INSTALL_PATH=${HOME}' >> ${BASH_ENV}
+
+  build_image: &build_image
+    run:
+      name: Build Docker Image
+      command: |
+        docker login quay.io -u="${quay_robot}" -p="${quay_token}"
+        docker-pull -f deploy/production.config
+        docker-build -f deploy/production.config
+
+  deploy: &deploy
+    run:
+      name: Deploy to Production
+      command: |
+        docker-push -f deploy/production.config
+        prepare-kubectl
+        k8s-deploy-and-verify -f deploy/production.config
+
+jobs:
+  build_image:
+    docker:
+      - image: quay.io/reactiveops/ci-images:v8-stretch
```

Comment:
In general we suggest pinning the minor version due to the versioning changes in v8.0.0, so I would rather this example show v8.0-stretch instead of v8-stretch.

Reply:
👍